### DISCUSSION TITLE
/Trust outside of claims

### DISCUSSION THREAD

**Discussion by Smaaaae** [GriefPrevention/GriefPrevention#1978] [Q&A]
### Observed Behavior

When you /trust outside of a claim, you are prompted with a message that the player has access to ALL your claims. 
Recently when using this command it only trusts in claims and not within subdivisions - I would like some clarity on whether this is a bug or a built in thing with the plugin.

### Expected Behavior

gives the trusted player access to every single part of every one of your claims

### Reproduction steps

stand in an unclaimed zone
/trust <player>
go back to claim check /trustlist
then stand in a subdivision and check /trustlist (says no one is trusted)

### Server version

```shell
1.18
```

### GriefPrevention version

```shell
no idea ! sorry - must be a recent one
```

### Configuration

```yaml
I don't know what this is - I am an staff member requesting this info for the owner so I don't have this data
```

### Running without GriefPrevention

- [ ] I attempted running the server without GriefPrevention installed.
- [ ] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [ ] I attempted running only GriefPrevention on the server.
- [ ] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [ ] I attempted testing for the issue on a new server.
- [ ] The issue still occurs on a new server.

### Using unmodified client

- [ ] I attempted testing for the issue with the vanilla client.
- [ ] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [X] I searched for similar issues before submitting this bug report.

**Comment by Jikoo** [comment]
Subclaims inherit all permissions from the parent unless specifically excluded via `/RestrictSubclaim`, so this does indirectly add permission to most subclaims. It is not a bug that unrestricted subclaims do not have redundant users added to their trust list.

It is technically a bug that restricted subclaims do not also have the trust added, but imo that's a good feature - usually people creating restricted subclaims are GP powerusers who would not want to blindly trust others across all their finnicky small subclaims.
